lib/fs.js

```diff
@@ -1813,11 +1843,25 @@ fs.realpath = function realpath(path, options, callback) {
   } else if (typeof options !== 'object') {
     throw new TypeError('"options" must be a string or an object');
   }
-  callback = makeCallback(callback);
```

Comment:
`makeCallback` comment says "Only use this function for callbacks that are passed to the binding layer, callbacks that are invoked from JS already run in the proper scope.". I think we are good here.